xtask/src/runwasm.rs

```diff
@@ -0,0 +1,123 @@
+use std::env;
+use std::process::Command;
+
+use crate::runchecks::{cargo_install, is_binary_installed, rustup};
+
+// Targets constants
+const WASM32_TARGET: &str = "wasm32-unknown-unknown";
+
+fn run_wasm_pack(rustflags: &str) -> Command {
+    // Set RUSTFLAGS parameters
+    env::set_var("RUSTFLAGS", rustflags);
+
+    // Build wasm-pack command
+    let mut command = Command::new("wasm-pack");
```

Comment:
```suggestion
    let mut command = Command::new("wasm-pack");
    command.current_dir("./examples/image-classification-web");
```

I think adding `current_dir` is important, since that in turn sets which `Cargo.toml` to use. If you add

```rust
    let exit = build_command
        .spawn()
        .unwrap()
        .wait()
        .unwrap();
    log::info!("build's spawn's exit: {}", exit);
```

to `dist` below you'll observe that it runs successfully with exit code 0. Unfortunately, this doesn't solve the larger issue with `arg.base.build_command(build_command)`. If you don't set `current_dir`, you'll observe that `build_command.spawn()` exits with the same error output as `arg.base.build_command(build_command)`. This makes me think that `arg.base.build_command(build_command)` isn't respecting `command.current_dir`.

Grepping `xtask-wasm` for `current_dir` yields only [one hit](https://github.com/rustminded/xtask-wasm/blob/7a39cfa31e6989479eb50e9e3a8f064549bcdc86/src/dist.rs#L188-L190), so I think it's an upstream problem.